### PR TITLE
fix:Explore Search: No title bar for item, non-existent menu for item, wrongly-worded menu for category 

### LINKED ISSUE #4271: Explore Search: No title bar for item, non-existent menu for item, wrongly-worded menu for category
Explore>Search>Items>item lacks a title (maybe white on white?) and three-dot menu containing "View item page":
adb

It should be like the equivalent category (notice the title "Rabbits" and the three-dot menu containing "View file page"):
adb

By the way, for categories "View file page" should be specifically reworded to "View category page".

To see these screens, go to Explore, tap in the search bar, type rabbit, tap either "categories" or "items", then tap the first row

### PR DESCRIPTION
**Pull request by Prince-kushwaha**
**Description (required)**

Fixes #4271 

What changes did you make and why?

* add toolbar in WikidataItemDetailsActivity
* minor change 
 * add javadoc

**Tests performed (required)**

Tested { ProdDebug} on {name of device :Redmi 4} with API level {25,28}.

**Screenshots (for UI changes only)**
Screenshot_2021-02-24-10-13-45-864_fr free nrw commons

Screenshot_2021-02-24-10-14-28-861_fr free nrw commons

### CODE REVIEW

**Comment by Prince-kushwaha** [comment]
@nicolas-raoul rewiew my this PR

**Comment by Prince-kushwaha** [comment]
up for review @nicolas-raoul , @neslihanturan

File: app/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -711,5 +711,7 @@ Upload your first media by tapping on the add button.</string>
   <string name="depicts_step_title">Depicts</string>
   <string name="license_step_title">Media License</string>
   <string name="media_detail_step_title">Media Details</string>
+  <string name="menu_view_category_page">View category page</string>
+  <string name="view_item_page">View item page</string>
```
Comment (neslihanturan) [inline app/src/main/res/values/strings.xml:715]: What about renaming "view_item_page" to "menu_view_item_page", just to be consistent with the string above ("menu_view_category_page") ?

**Review by neslihanturan — CHANGES_REQUESTED** [review]

**Comment by neslihanturan** [comment]
Hey @Prince-kushwaha , thanks for the PR. There are 3 issues in the issue linked (ideally they should have been separated issues). 
1- The text "view file page" should be "View category page". (PR solves this)
2- Item should have a tree dot menu containing "View item page" (PR solves this)
3- Adds toolbar to media item when it is selected from search->item (PR solves this)

So I am good to merge this PR as soon as the minor string naming issue is solved, thanks :)

File: app/src/main/java/fr/free/nrw/commons/explore/depictions/WikidataItemDetailsActivity.java [hunk 2]
```diff
@@ -188,4 +198,34 @@ public static void startYourself(Context context, DepictedItem depictedItem) {
         intent.putExtra("entityId", depictedItem.getId());
         context.startActivity(intent);
     }
+
+    /**
+     * this function inflate the menu
```
Comment (nicolas-raoul) [inline app/src/main/java/fr/free/nrw/commons/explore/depictions/WikidataItemDetailsActivity.java:203]: this -> This (first letter uppercase in javadoc)
inflate -> inflates

File: app/src/main/java/fr/free/nrw/commons/explore/depictions/WikidataItemDetailsActivity.java [hunk 3]
```diff
@@ -188,4 +198,34 @@ public static void startYourself(Context context, DepictedItem depictedItem) {
...
+    @Override
+    public boolean onCreateOptionsMenu(Menu menu) {
+        MenuInflater menuInflater=getMenuInflater();
+        menuInflater.inflate(R.menu.menu_wikidata_item,menu);
+        return super.onCreateOptionsMenu(menu);
+    }
+
+    /**
+     * This method handles the logic on ItemSelect in toolbar menu
+     * Currently only 1 choice is available to open Wikidataitem details page in browser
```
Comment (nicolas-raoul) [inline app/src/main/java/fr/free/nrw/commons/explore/depictions/WikidataItemDetailsActivity.java:214]: Wikidataitem -> Wikidata item

**Review by nicolas-raoul — CHANGES_REQUESTED** [review]
Great PR, it works perfectly for all 3 points, as Neslihan mentioned.
I found a few minor details worth fixing.

**Review by neslihanturan — APPROVED** [review]
Thanks for the cooperation @Prince-kushwaha :)

**Pull request merged**